Changes

```diff
@@ -18,6 +18,10 @@ Next version (tbd):
   a short description in overviews.
   (Florian Angeletti)
 
+- PR#7351: ocamldoc, use semantic tags rather than <br> tags in
+  the html backend
+  (Florian Angeletti, request and review by Daniel Bunzli )
```

Comment:
"Bünzli" would be the correct spelling. (I now realize that @alainfrisch also spelled it wrong a couple of time, you could fix those occurrences.)